src/core/services/theming/theming.js

```diff
@@ -282,6 +357,8 @@ function ThemingProvider($mdColorPalette) {
       alwaysWatchTheme = alwaysWatch;
     },
 
+    enableBrowserColor: enableBrowserColor,
+
```

Comment:
I think it might be helpful to go ahead and provide/document the `setBrowserColor()` method here too. I can see some people just wanting to set it to white/black and it being cumbersome to figure out the proper theme/palette/hue.